curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/authentication-with-socket.io.md

```diff
@@ -33,7 +33,11 @@ io.use(
 );
 ```
 
-Be sure to add the `key` and `store` to the `session` middleware mounted on the app. This is necessary to tell *SocketIO* which session to relate to.
+Note that configuring Passport authentication for Socket.IO is very similar to the way we configured the `session` middleware for the API. This is because they are meant to use the same authentication method — get the session id from a cookie and validate it.
+
+Before when we configured the `session` middleware we didn't set the cookie name for session (`key`) explicitly. This is because the `session` package was using the default value. But now that we've added another package which needs access to the same value from the cookies, we need to set the `key` value explicitly in both configuration objects.
```

Comment:
```suggestion
Previously, when we configured the `session` middleware, we didn't explicitly set the cookie name for session (`key`). This is because the `session` package was using the default value. Now that we've added another package which needs access to the same value from the cookies, we need to explicitly set the `key` value in both configuration objects.
```